src/gui/gui.pri

```diff
@@ -6,28 +6,28 @@ include(properties/properties.pri)
 unix:!macx:dbus: include(qtnotify/qtnotify.pri)
 
 HEADERS += \
-    $$PWD/about_imp.h \
+    $$PWD/aboutdialog.h \
     $$PWD/addnewtorrentdialog.h \
     $$PWD/advancedsettings.h \
     $$PWD/autoexpandabledialog.h \
-    $$PWD/banlistoptions.h \
+    $$PWD/banlistoptionsdialog.h \
     $$PWD/categoryfiltermodel.h \
     $$PWD/categoryfilterproxymodel.h \
     $$PWD/categoryfilterwidget.h \
     $$PWD/cookiesdialog.h \
     $$PWD/cookiesmodel.h \
-    $$PWD/deletionconfirmationdlg.h \
+    $$PWD/deletionconfirmationdialog.h \
     $$PWD/downloadfromurldialog.h \
-    $$PWD/executionlog.h \
+    $$PWD/executionlogwidget.h \
     $$PWD/fspathedit.h \
     $$PWD/fspathedit_p.h \
     $$PWD/guiiconprovider.h \
     $$PWD/hidabletabwidget.h \
     $$PWD/ipsubnetwhitelistoptionsdialog.h \
     $$PWD/loglistwidget.h \
     $$PWD/mainwindow.h \
-    $$PWD/messageboxraised.h \
-    $$PWD/optionsdlg.h \
+    $$PWD/messageboxraiseddialog.h \
```

Comment:
It should have messagebox suffix (MessageBox for appropriate class) since it is derived from QMessageBox and inherits its behavior.

Reply:
So... just messagebox then?

Reply:
RaisedMessageBox